### ISSUE TITLE
US 9 - Seller's selling list

### ISSUE THREAD

**Issue by DavevonH** [Rick1Marques/star-wars-bazar-app#9]
## Value proposition

As a user,
I want to view the items that a seller has listed,
so that I can explore more products from that seller and potentially make a purchase

## Description

![Image](https://user-images.githubusercontent.com/132745980/268938129-ada2ed65-ea1f-47f8-9a1b-38761ee9936b.png)

## Acceptance criteria

- [ ] On the "Starship Details" page the user can click on the sellers' image
   which links to the "sellers' selling list" page 

- [ ] The "sellers' selling list" page shows the following information:
    - name of the seller
    - an image of the seller
    - a list of starships the seller is offering

- [ ] Each starship on the list shows:
    - the name of the starship
    - image of the starship
    - price
    - buy button as a placeholder

## Tasks

- [ ] List the tasks that need to be performed within the dev team

**Comment by ahohnsen** [comment]
Great 🙌🏼 

I have just two little suggestions:

1. I would rephrase the value proposition a bit. The second line should mention the feature 👉🏼 in this case the feature is, that the user can see all items another user is selling. I suggest something like:

_As a user,
I want to view the items that a seller has listed,
so that I can explore more products from that seller and potentially make a purchase._

2. Would it be possible that a seller is not actively selling any items at the moment? 🤔  In such cases, a message like "This user does not have any items listed for sale currently" could be displayed instead of an empty page.

**Comment by shrocket** [comment]
Thank you for letting us look at your app! Here are our comments:
1. Please change the word "Bazar" to "Bazaar"
2. Please change the word "Händler" to "Dealer" (and be on the lookout for any more "Denglisch"!)
3. Please add a "Buy" button (as per acceptance criteria)
4. Please add a link back to the "Starship Details" page (as per acceptance criteria)

Additionally, why is the `homepage` text only and unreadable (because color/contrast) at that?

**Comment by Rick1Marques** [comment]
As we said on the code review, there are things to change on the backend, that we will do later this week.
The Buy button its part of another story and we will render just a placeholder for this one.
We removed the back link for now.
